tst/fakepkg/README.md

```diff
@@ -1,3 +1,20 @@
-The GAP 4 package `fakepkg'
-==============================
+The GAP 4 package `fakepkg`
+===========================
+
+This is a mock package to be used to test GAP library code
+related to GAP packages, for example to validate `PackageInfo.g`
+files.
+
+It also has a manual written in `gapmacro.tex` format. This
+format is obsolete, being superseded by GAPDoc; however, it
+is still used by some packages, and hence we would like to
+ensure that we are still able to render their documentation
+using the "screen" online help viewer. This is why we use 
+this manual to test for the code from `lib/helpt2t.g{d,i}`
+which converts `gapmacro.tex`-based manuals into plain text.
```

Comment:
BTW, we could *also* add a GAPDoc manual to the mock package -- a package can have more than one help book, after all. (This is not a request for this PR, just wanted to point this out)